Disable exported function definitions because they're a giant pointless bug.

diff -ru bash/variables.c bash.bak/variables.c
--- bash/variables.c	2002-06-25 08:43:33.000000000 -0500
+++ bash.bak/variables.c	2014-09-28 22:04:31.497654646 -0500
@@ -260,7 +260,7 @@
 
       /* If exported function, define it now.  Don't import functions from
 	 the environment in privileged mode. */
-      if (privmode == 0 && read_but_dont_execute == 0 && STREQN ("() {", string, 4))
+      if (0)
 	{
 	  string_length = strlen (string);
 	  temp_string = (char *)xmalloc (3 + string_length + char_index);
